code/lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -369,7 +370,14 @@ export class StoryIndexGenerator {
       };
       return docsEntry;
     } catch (err) {
-      logger.warn(`🚨 Extraction error on ${relativePath}: ${err}`);
+      logger.warn(`🚨 Extraction error on ${chalk.blue(relativePath)}: ${err}`);
+      if (err.source?.match(/mdast-util-mdx-jsx/g)) {
+        logger.warn(
+          `💡 This seems to be an MDX2 syntax error. Please refer to the following resource for assistance on how to fix this: ${chalk.yellow(
+            'http://link-to-resource/here'
```

Comment:
TODO: to be replaced when we actually have something

Reply:
https://storybook.js.org/migration-guides/7.0 this one?